Demo.java

```diff
@@ -0,0 +1,78 @@
+public class Demo {
+
+    static Participant[] prtcp;
+    static Obstacle[] obstc;
+    static int y; // participants count
```

Comment:
do not name variables like that! What is the reason of setting a name 'y' to this variable and then writing a comment for it to describe the meaning? Wouldn't it be better to rename `y` to `participantsCount`?